### PR TITLE
MISUV-8274b Test new ispec with old http client

### PR DESCRIPTION
**Pull request by mywyau**
Passes against old httpclient when running the command

sbt "it/testOnly *ITSAStatusUpdateConnectorISpec*"

### CODE REVIEW

**Comment by mywyau** [comment]
Screenshot 2024-10-01 at 10 31 52

**Comment by mywyau** [comment]
Screenshot 2024-10-01 at 11 06 21

**Comment by mywyau** [comment]
Branch still uses old client

Screenshot 2024-10-01 at 11 07 16

File: it/test/connectors/ITSAStatusUpdateConnectorISpec.scala [hunk 1]
```diff
@@ -0,0 +1,363 @@
...
+
+    ".makeITSAStatusUpdate()" when {
+
+      "request body" should {
+
+        "be built correctly for OptOutReason" in {
+
+          WiremockHelper.stubPut(s"/income-tax-view-change/itsa-status/update/$taxableEntityId", NO_CONTENT, correctOptOutRequestBody, "{}")
+
+          val result = connector.makeITSAStatusUpdate(taxYear2024, taxableEntityId, optOutUpdateReason).futureValue
```
Comment (ian-hmrc) [inline it/test/connectors/ITSAStatusUpdateConnectorISpec.scala:41]: Is it possible to treat `makeITSAStatusUpdate(...)` as private and test via the `optOut(...)` and `optIn(...)` functions?

**Pull request closed without merging**